lib/tasks/schools_data.rake

```diff
@@ -0,0 +1,29 @@
+require "rake"
+
+namespace :schools_data do
+  desc "Import schools data from Get Information About Schools"
+  task import: :environment do
+    SchoolDataImporterJob.perform_later
+  end
+
+  desc "Export schools eligible for Student loans to csv"
+  task "export:student_loans": :environment do
+    export_school_data(:student_loans)
+  end
+
+  desc "Export schools eligible for Maths and physics to csv"
+  task "export:maths_and_physics": :environment do
+    export_school_data(:maths_and_physics)
+  end
+end
+
+def export_school_data(policy_name)
+  require "csv"
```

Comment:
It's more conventional to stick this at the top of the file rather than here.

Reply:
bah! I moved it down! :(